docs/guide/quickstart.rst

```diff
@@ -0,0 +1,292 @@
+    ``Intents`` are the method to specify which gateway events you wish to receive. ``Intents.default()`` means you will receive all events that are not locked behind bot verification.
+    For more information, see :ref:`intents_primer`.
+
+.. code-block:: python
+
+    @bot.listen()
+    async def on_ready():
+        print(f"Ready! I am {bot.user} and my ID is {bot.user.id}")
+
+This is the ``ready`` event. It is called when the bot has finished loading and everything is cached.
+We use the :meth:`@bot.listen() <ext.commands.Bot.listen>` decorator as to not override the main event.
+For a list of available events, see :ref:`discord-api-events`.
+
+.. _guide_quickstart_bot_event_warning:
+
+.. warning::
+
+    Using :meth:`@bot.event <ext.commands.Bot.event>` is also valid, but keep in mind that using :meth:`@bot.event <ext.commands.Bot.event>` with the ``on_message`` event can cause your commands to stop responding.
+    To remedy this, either switch to :meth:`@bot.listen() <ext.commands.Bot.listen>`, or add :meth:`~ext.commands.Bot.process_commands` to your ``on_message`` event:
+
+    .. code-block:: python
+
+        @bot.event
+        async def on_message(message: discord.Message):
+            print(f'Received "{message.clean_content}" from {message.author}')
+            # IMPORTANT:
+            await bot.process_commands(message)
+
+.. warning::
+
+    ``on_ready`` can and will be called multiple times throughout your bot's uptime. You should avoid doing any kind of state-management here, such as connecting and loading your database.
+
+.. code-block:: python
+
+    @bot.command()
+    async def echo(ctx: commands.Context, *, sentence: commands.clean_content):
+        await ctx.send(sentence)
+
+Here's where our commands will be defined. We use the :meth:`@bot.command() <ext.commands.Bot.command>` decorator to flag this function as a command.
+This creates a command ``!echo`` that we can type into a channel, and the bot will respond with the given sentence. A few key features:
+
+- :class:`~ext.commands.Context` refers to the command invocation context - this includes the :attr:`~ext.commands.Context.channel`, command :attr:`~ext.commands.Context.author`, :attr:`~ext.commands.Context.message` and more.
+- The ``*`` is a sign to tell discord.py that the following parameter should **consume all text afterward** and condense it into that parameter.
+    .. note::
+    
+        As this consumes all text, parameters defined **after the next** will never be filled, so you should never have more than one parameter after the ``*``.
+
+- :meth:`~ext.commands.clean_content` is a :ref:`Converter <ext_commands_api_converters>` used to clean user pings and ``@everyone`` mentions. This way people will not be able to ping everyone by using your bot. For more information, see :ref:`ext_commands_commands_converters`.
+
+.. code-block:: python
+
+    TOKEN = "your bot token here"
+    bot.run(TOKEN)
```

Comment:
Do we show off the new async changes alongside `Bot.run`? E.g. our new asyncio.run methods.

Reply:
I think it’s a good idea to show both.